examples/test-error.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
```

Comment:
pls revert the 3 example files. or merge them into one and name it analytics-user-error.amp.html